src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java

```diff
@@ -284,14 +283,6 @@ private static void validateTableAccess(Client engineClient, JSONObject mapping,
             } catch (DataServiceException e) {
                 databaseTableErrorList.add(String.format("Couldn't validate access to Database '%s' Table '%s', with exception '%s'", database, table, ExceptionUtils.getStackTrace(e)));
             }
-            if (hasAccess) {
```

Comment:
if mappingName is specified then Line No 290 will be executed and the mappingName will be validated.
if mappingName is not specified then Line No 269 : mapping.optString(MAPPING) will return "" and it will not enter Line 290 if (hasAccess && StringUtils.isNotEmpty(mappingName)).